### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by odaialazzeh**
## HTML pages:
- [ ] We have created an index page for the project with the navigation bar.

## CSS file:

- [ ] We have created style.css which contains the styles for the index page.

## JavaScript file:

- [ ] We created a javascript file with codes to add and delete books, and to direct to specific parts.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,19 +8,55 @@
...
+        </p>
+        <ul>
+            <li>Our e-mail: mail@mail.com</li>
+            <li>Our phone number : 00</li>
+            <li>Our address: Streetname: , City: , Country</li>
+        </ul>
+    </div>
+
+
+    <footer>
```
Comment (topeogunleye) [inline index.html:54]: - [ ] I kindly suggest that you use css property of `position: fixed` to fix your footer to the bottom of the page

https://www.loom.com/share/d82e28c6c38b4d1ea54908dc2ca7bcdb

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @odaialazzeh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GiuseppeTG — APPROVED** [review]
Hi @odaialazzeh  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 : 
Congratulations! 🎉 

Alt Text

## Highlights

✅ App works nice
✅ Implemented changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GiuseppeTG** [comment]
- [OPTIONAL] Please consider naming your branches with a more descriptive name. Knowing what is being implemented in each branch by its name is nice. For example, if you are doing just style in a branch, the branch name can be styles. 🤓 
You don't have to change this, just keep it in mind for future projects.

**Pull request merged**